tests/core/index_management/test_services_xml.py

```diff
@@ -200,6 +200,67 @@ def test_config_components(self):
                 """
         self._assertStringsEqualIgnoringWhitespace(expected_xml, service_xml.to_xml())
 
+    def test_config_components_should_preserve_document_processing_nodes(self):
```

Comment:
Add a comment as to why we're testing this

Reply:
Will do

Reply:
Just to confirm I should just add a generic comment instead of referring to the Cloud issue, right?